tests/unit/modules/UploadModule.spec.ts

```diff
@@ -107,10 +104,20 @@ describe('UploadModule.ts', () => {
     expect(uploadModule.filename).toEqual('')
     expect(uploadModule.originalFilename).toEqual('')
     expect(uploadModule.title).toEqual('')
-    expect(uploadModule.hash).toEqual('')
     expect(uploadModule.description).toEqual('')
     expect(uploadModule.tags).toEqual([])
     expect(uploadModule.md5).toEqual('')
     expect(uploadModule.status).toBe(ProcessStatus.NotReady)
   })
+
+  it ('correctly generates hashes', () => {
+    const metaMaskModule = getModule(MetaMaskModule, appStore)
+    metaMaskModule.setPublicWalletAddress('address')
+    const uploadModule = getModule(UploadModule, appStore)
+    uploadModule.setTitle('title')
+    expect(uploadModule.hash).toEqual('0x52a6ac5d523467bd8af28d0067f695048277b504b34ae96525f5c3300644d5cd')
```

Comment:
ugh. hard coded hash... likely better to just check the `.length` as any keccak is guaranteed in len...

Reply:
you can also test verses the Datatrust contract in .js
`datatrust.getHash(str)` will return a keccak

Reply:
calling the Datatrust contract in the spec is a call too far for me.  expecting len > 2 and testing `startsWith`